bindings/python/examples/simple_transaction/request_funds.py

```diff
@@ -20,8 +28,10 @@
 print(
     f'balance before faucet request: { balance[ "baseCoin" ][ "available" ] }')
 
+addresses = account.generate_addresses(1)
```

Comment:
Why not get the existing address?

Reply:
Ser. I'm again just copying existing examples around 😆

Reply:
You copy the wrong parts :trollface: